app/scripts/controllers/projects.js

```diff
@@ -16,14 +16,50 @@ angular.module('openshiftConsole')
     $scope.showGetStarted = false;
     $scope.canCreate = undefined;
 
+    var sortProjects = function() {
```

Comment:
We are on angular-patternfly 3.5.1, i think this todo has aged & can be removed.